src/formBuilder/FormBuilder.js

```diff
@@ -167,52 +169,12 @@ const useStyles = createUseStyles({
       maxHeight: '500px',
       overflowY: 'scroll',
     },
-    '& .card-container': {
-      width: '70%',
-      minWidth: '400px',
-      margin: '2em auto',
-      border: '1px solid var(--gray)',
-      borderRadius: '4px',
-      backgroundColor: 'white',
-      '& h4': {
-        width: '100%',
-        textAlign: ['left', 'left'],
-        display: 'inline-block',
-        color: '#138ac2',
-        margin: '0.25em 0.5em 0 0.5em',
-        fontSize: '18px',
-      },
-      '& .d-flex': { borderBottom: '1px solid var(--gray)' },
-      '& .label': { cssFloat: 'left' },
-    },
-    '& .card-container:hover': { border: '1px solid var(--green)' },
-    '& .card-dependent': { border: '1px dashed var(--gray)' },
     '& .card-add': {
       cursor: 'pointer',
       display: 'block',
       color: '$green',
       fontSize: '1.5em',
     },
-    '& .section-container': {
-      width: '90%',
-      minWidth: '400px',
-      margin: '2em auto',
-      border: '1px solid var(--gray)',
-      borderRadius: '4px',
-      '& h4': {
-        width: '100%',
-        textAlign: ['left', 'left'],
-        display: 'inline-block',
-        color: '#138ac2',
-        margin: '0.25em 0.5em 0 0.5em',
-        fontSize: '18px',
-      },
-      '& .d-flex': { borderBottom: '1px solid var(--gray)' },
-      '& .label': { cssFloat: 'left' },
-    },
-    '& .section-container:hover': { border: '1px solid var(--green)' },
-    '& .section-dependent': { border: '1px dashed var(--gray)' },
-    '& .section-requirements': { border: '1px dashed black' },
```

Comment:
None of these rules are needed anymore? Do we get them for free in the reactstrap upgrade?

Reply:
they seem  to be duplicative, and one of them was causing issues with the reactstrap upgrade (the var(--gray) thing) so i figured best to just remove